activerecord-postgis-adapter.gemspec

```
@@ -19,13 +19,13 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = ">= 2.7.0"
 
-  spec.add_dependency "activerecord", "~> 7.0.0"
+  spec.add_dependency "activerecord", "~> 7.1.beta1"
```

```suggestion
  spec.add_dependency "activerecord", ">= 7.0", "< 7.2"
```

we still do support AR 7.0.8 don't we? @keithdoggett I don't know how we handled that in the past but it seems to me that we are quite strict, aren't we?

---

We bumped major version at every AR  major version. It on the readme.

Also API are not compatible fully.

---

Two things though:

- this is not a major version
- shouldn't we care about compatibility for our users ?

---

In rails, that a major version .
7.0 to 7.1 is major since it has breaking changes.

---

The user api in this adapter is unchanged. 

Users can updated it freely and the others small changes were just throwing warnings in ruby 3.2